main.cpp

```diff
@@ -334,12 +464,92 @@ void TrainingPlan::createPlan ( int intensity, int rounds, double restPeriod )
         shoulderPress.doExercise( trainee );
         deadLift.doExercise( trainee );
         squat.doExercise( trainee );
+        std::cout << "Round Complete.\n\n";
     }
 }
 
 
 int main()
 {
     Example::main();
+
+//1
+    std::cout << "\n**=============================**\n\n";
```

Comment:
- [ ] Write a free function to print this out, and replace all instances of this `std::cout` call in `main()` with a call to that free function;